---
bool botao1
bool botao2
bool botao3